src/OpenTelemetry.Exporter.Prometheus.HttpListener/OpenTelemetry.Exporter.Prometheus.HttpListener.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- OmniSharp/VS Code requires TargetFrameworks to be in descending order for IntelliSense and analysis. -->
+    <TargetFrameworks>netcoreapp3.1;net462</TargetFrameworks>
```

Comment:
Might be able to be `netstandard2.0;net462` or _maybe_ even just `netstandard2.0`?

Reply:
yes, I'm guessing using netstandard2.0 should be sufficient here.

Reply:
I stumbled across the Castle.Core dependency issue while using netstandard2.0. Will do more digging and report back if I can fix it.

Reply:
Updated to be using netstandard2.0;net462 here: https://github.com/open-telemetry/opentelemetry-dotnet/pull/3430